course_inventory/inventory.py

```diff
@@ -134,13 +134,13 @@ def gather_course_data_from_api(account_id: int, term_id: int) -> pd.DataFrame:
 
 # Function(s) - UDW
 
-def process_sis_id(orig_sis_id: str) -> Union[int, None]:
+def process_sis_id(orig_sis_id: str) -> Optional[int]:
```

Comment:
I considered using `Optional` in some of my code, but it made it look like a result value from the function was, well… optional.  I preferred `Union[…, None]` because it made it explicit that my function would return either the first type or `None`.

Reply:
I can try switching back to that. I like `Union`, too.

Reply:
Done, I made the switch back here.